pkg/ibmvpc/parser.go

```diff
@@ -1030,7 +1030,7 @@ func printConfig(c *vpcmodel.VPCConfig) {
 		if n.IsExternal() {
 			continue
 		}
-		fmt.Println(strings.Join([]string{n.Kind(), n.Cidr(), n.Name(), n.UID()}, separator))
+		fmt.Println(strings.Join([]string{n.Kind(), n.CidrOrAddress(), n.Name(), n.UID()}, separator))
```

Comment:
same comment...

Reply:
This is just a printing function, so I don't find it too confusing here. 
From the function documentation it is known that it returns an address for internal node, which is the case here.

Reply:
yes, this is an example of a proper use of the function